Makefile

```diff
@@ -532,3 +535,26 @@ endif
 	bin/manager --object-deletion-protection=false --log-level=debug \
 	--atlas-domain=$(MCLI_OPS_MANAGER_URL) \
 	--global-api-secret-name=mongodb-atlas-operator-api-key
+
+.PHONY: local-docker-build
+local-docker-build:
+	docker build -t $(LOCAL_IMAGE) .
+
+.PHONY: yq
+yq:
+	go install github.com/mikefarah/yq/v4@latest
```

Comment:
test using `which` whether this is already installed.

Reply:
Done